uber/templates/registration/check_in_form_base.html

```diff
@@ -33,12 +33,24 @@ <h4>{{ attendee.full_name }} currently owes <strong>${{ '%0.2f' % attendee.amoun
         $button.attr('disabled', 'disabled');
     }
 };
-var loadCheckInForm = function () {
+loadCheckInForm = function () {
+    $('#stripeModal').off('hidden.bs.modal');
```

Comment:
All this nonsense is to make sure closing the Stripe modal on purpose cancels a payment, but when the Stripe modal closes because of a payment completion it DOESN'T cancel the payment, and also the modal needs to be removed from the page entirely because Stripe Elements gets confused if you leave it lying around, and...

... I love JavaScript.